packages/vite/src/node/server/__tests__/pluginContainer.spec.ts

```diff
@@ -106,13 +146,21 @@ async function getPluginContainer(
   inlineConfig?: UserConfig
 ): Promise<PluginContainer> {
   const config = await resolveConfig(
-    { configFile: false, ...inlineConfig },
+    {
+      configFile: false,
+      server: { preTransformRequests: false },
```

Comment:
Is this specifically required by the test you added, or can it be moved to another PR?

Reply:
I answered this comment already (maybe GitHub has lost it?).  The answer is yes, it is.  But those tests _can_ be done differently and moved to another PR.